example/undefinedValue.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const Errors = require('../Errors.js');
+
+const importantPassword = process.env.IMPORTANT_PASSWORD;
+
+function displayImportantPassword(){
+    if(!importantPassword){
+        const err = new Errors.UndefinedValue;
+        err.setMessage(
+            'importantPassword',
+            importantPassword
+        );
+        throw err;
+    }
+    console.log(`Dont forget this importantPassword: ${importantPassword}, don't!!!`);
+}
```

Comment:
I think the console log is confusing for the example. Makes me feel like that is what it logs when the error happens.